phpBB/phpbb/db/migration/data/v32x/user_notifications_table_unique_index.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\migration\data\v32x;
+
+class user_notifications_table_unique_index extends \phpbb\db\migration\migration
+{
+	static public function depends_on()
+	{
+		return array(
+			'\phpbb\db\migration\data\v32x\user_notifications_table_remove_duplicates',
+		);
+	}
+
+	public function update_schema()
+	{
+		return array(
+			'add_unique_index' => array(
+				$this->table_prefix . 'user_notifications' => array(
+					'itm_type_itm_id_usr_id_method'	=> array('item_type', 'item_id', 'user_id', 'method'),
```

Comment:
I think this would be too long on *any* setup since even the length of the index itself as already at 29 chars.